platform/core/src/utils/measurementGroupings.ts

```diff
@@ -0,0 +1,13 @@
+export const groupByStudy = displaySetService => (groupedMeasurements, item) => {
```

Comment:
Just add a docs to indicate that this groups measurements by study in order to allow display and saving by study.